### PR TITLE
[1LP][RFR][NOTEST] Added new manual test cases

### PR DESCRIPTION
**Pull request by ganeshhubale**
Added manual test cases for BZs:
1715396, 1716443 in automate FA

### CODE REVIEW

File: cfme/tests/automate/test_automate_manual.py [hunk 1]
```diff
@@ -902,3 +902,75 @@ def test_retire_vm_with_vm_user_role():
...
+@pytest.mark.tier(1)
+def test_dialog_element_values_passed_to_button():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        startsin: 5.10
+        casecomponent: Automate
+        testSteps:
+            1. Import the attached BZ automate domain and dialog exports. The automate domain
```
Comment (john-dupuy) [inline cfme/tests/automate/test_automate_manual.py]: Can you reference the BZ that has the attachment here? 1715396

File: cfme/tests/automate/test_automate_manual.py [hunk 2]
```diff
@@ -902,3 +902,75 @@ def test_retire_vm_with_vm_user_role():
...
+    pass
+
+
+@pytest.mark.tier(1)
+def test_git_refresh_with_renamed_yaml():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: negative
```
Comment (john-dupuy) [inline cfme/tests/automate/test_automate_manual.py]: Q: Isn't this a positive test case?

Comment (ganeshhubale) [reply cfme/tests/automate/test_automate_manual.py]: Yeah. I agree as we are checking successful refresh of domain.

File: cfme/tests/automate/test_automate_manual.py [hunk 3]
```diff
@@ -902,3 +902,75 @@ def test_retire_vm_with_vm_user_role():
...
+    pass
+
+
+@pytest.mark.tier(1)
+def test_git_refresh_with_renamed_yaml():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
```
Comment (lina-nikiforova) [inline cfme/tests/automate/test_automate_manual.py]: Optional: it is positive by default
https://github.com/ManageIQ/integration_tests/blob/master/conf/polarion_tools.yaml#L9

**Pull request merged**